api/AbortController.json

```diff
@@ -32,10 +32,18 @@
             "version_added": "53"
           },
           "safari": {
-            "version_added": "11.1"
+            "version_added": "11.1",
+            "partial_implementation": true,
+            "notes": [
+              "As of version 12, even though <code>window.AbortController</code> is defined, it <a href='https://bugs.webkit.org/show_bug.cgi?id=174980'>doesn't</a> really abort <code>fetch</code> requests. See ticket</a>"
```

Comment:
I'd suggest a couple changes here:

* We can drop "As of version 12" since it looks like Safari has not yet implemented the right behavior in any version
* I'd change the last bit to be `See <a href='https://bugs.webkit.org/show_bug.cgi?id=174980'>bug 174980</a>.` and remove the other link to be more consistent with other notes

Reply:
Sounds good!